website/src/components/SearchPage/Table.tsx

```diff
@@ -145,6 +146,9 @@ export const Table: FC<TableProps> = ({
                                     key={c.field}
                                     onClick={() => handleSort(c.field)}
                                     className='px-2 py-3 text-xs font-medium tracking-wider text-gray-500 uppercase cursor-pointer last:pr-6 text-left'
+                                    style={{
+                                        minWidth: c.columnWidth != null ? `${c.columnWidth}px` : undefined,
+                                    }}
```

Comment:
I was confused at first for a moment and wondered whether this was catching `c.columnWidth === undefined` until I saw that this was using `!=`. What would you think about storing `c.columnWidth !== null && c.columnWidth !== undefined ? `${c.columnWidth}px` : undefined` in a separate variable and then use it here and below?

Reply:
Ah, I hadn't particularly taken in that we were using `!=` as I did this with `aider`. Funny, I actually thought our linter didn't allow that!

Happy to go for your approach.